From fa8e6959303cb136029330f5991ff3721f3c6767 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 21 Aug 2019 14:48:43 +0800
Subject: [PATCH 2/4] clk: bcm2835: correct format with size_t type printk

drivers/clk/bcm/clk-bcm2835.c:2274:15: warning: format '%d' expects argument of type 'int',
	but argument 4 has type 'long unsigned int' [-Wformat=]
  dev_dbg(dev, "registered %d clocks\n", asize);
               ^~~~~~~~~~~~~~~~~~~~~~~~  ~~~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/clk/bcm/clk-bcm2835.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/bcm/clk-bcm2835.c b/drivers/clk/bcm/clk-bcm2835.c
index a142253afdff..4700c454b2f0 100644
--- a/drivers/clk/bcm/clk-bcm2835.c
+++ b/drivers/clk/bcm/clk-bcm2835.c
@@ -2271,7 +2271,7 @@ static int bcm2835_clk_probe(struct platform_device *pdev)
 		return ret;
 
 	/* note that we have registered all the clocks */
-	dev_dbg(dev, "registered %d clocks\n", asize);
+	dev_dbg(dev, "registered %zu clocks\n", asize);
 
 	return 0;
 }
-- 
2.17.1

